waspc/src/Wasp/Analyzer/Evaluator/Evaluation/TypedExpr/Combinators.hs

```diff
@@ -152,7 +153,9 @@ tuple4 eval1 eval2 eval3 eval4 = evaluation $ \(typeDefs, bindings) -> withCtx $
 -- | An evaluation that expects an "ExtImport".
 extImport :: TypedExprEvaluation AppSpec.ExtImport.ExtImport
 extImport = evaluation' . withCtx $ \ctx -> \case
-  TypedAST.ExtImport name file -> pure $ AppSpec.ExtImport.ExtImport name file
+  TypedAST.ExtImport name fileStr -> case SP.parseRelFileP fileStr of
```

Comment:
Here I parse file path from String into StrongPath.

This means that in Parser.AST, ExtImport uses String for the path, and same goes for TypeChecker.AST, but then in the final step of evaluation, we parse it from String into StrongPath, since we know it has to be relative and is a path to file. If parsing fails, EvaluationError is thrown.

We could have done it differently -> I could have instead parsed it immediately during Parser step, and then Parser.AST and TypeChecker.AST would both contain StrongPath instead of String/FilePath. If parsing fails, ParserError would be thrown.

I wasn't sure which way is better -> while it might sound like it is better to do it immediately, in practice I found that it requires more changes in the code than doing it later, while not, at least for now, offering any extra benefits, so I decided to go with simpler approach here. But, we could in the future decide to do this parsing earlier.